forge-gui/res/cardsfolder/l/long_list_of_the_ents.txt

```diff
@@ -0,0 +1,11 @@
+Name:Long List of the Ents
+ManaCost:G
+Types:Enchantment Saga
+K:Saga:6:DBNoteType,DBNoteType,DBNoteType,DBNoteType,DBNoteType,DBNoteType
+SVar:DBNoteType:DB$ ChooseType | Type$ Creature | Note$ True | SubAbility$ DBDelayTrig | SpellDescription$ Note a creature type that hasn't been noted for CARDNAME. When you cast your next creature spell of that type this turn, that creature enters the battlefield with an additional +1/+1 counter on it.
+SVar:DBDelayTrig:DB$ DelayedTrigger | Mode$ SpellCast | ValidCard$ Creature.NotedType | ThisTurn$ True | ValidActivatingPlayer$ You | Execute$ TrigEffect | TriggerDescription$ When you cast your next creature spell of that type this turn, that creature enters the battlefield with an additional +1/+1 counter on it.
```

Comment:
Like I said you'll need to write it like _Leori, Sparktouched Hunter_ because NotedType would include all previous ones

Reply:
Does the trigger still need notedtype instead of chosentype ?

Reply:
should work now hopefully

Reply:
I think it does now ! 
![image](https://github.com/Card-Forge/forge/assets/67333662/5d716dab-937a-4f05-adda-bad13c0946e5)